api/server/routes_create_update.go

```diff
@@ -32,14 +33,36 @@ func (s *Server) handleRoutesPostPutPatch(c *gin.Context) {
 		handleErrorResponse(c, err)
 		return
 	}
-	if method != http.MethodPatch {
-		err = s.ensureApp(ctx, &wroute, method)
-		if err != nil {
-			handleErrorResponse(c, err)
-			return
-		}
+	appName := c.MustGet(api.App).(string)
+
+	appID, err := s.ensureApp(ctx, appName, &wroute, method)
```

Comment:
this appears to add a bug that patch may now create an app. we should have a test case for that one, but perhaps we don't.

Reply:
i see, this is no longer for put, nvm then